runtime/doc/usr_04.txt

```diff
@@ -349,9 +350,6 @@ Notice that "yw" includes the white space after a word.  If you don't want
 this, use "ye".
 
 The "yy" command yanks a whole line, just like "dd" deletes a whole line.
-Unexpectedly, while "D" deletes from the cursor to the end of the line, "Y"
-works like "yy", it yanks the whole line.  Watch out for this inconsistency!
-Use "y$" to yank to the end of the line.
```

Comment:
Maybe this can be change to a mention about the `Y` command and its default mapping.

Reply:
I tried, but it's wonky. I don't know how to write a better beginner-friendly explanation. I'm open to ideas.